projects/elife.yaml

```diff
@@ -548,6 +548,29 @@ recommendations:
                     - bus-subjects--{instance}
                     - bus-collections--{instance}
 
+personalised-covers:
+    domain: False
+    subdomain: personalised-covers
+    repo: https://github.com/elifesciences/personalised-covers
+    formula-repo: https://github.com/elifesciences/personalised-covers-formula
+    formula-dependencies:
+        - https://github.com/elifesciences/builder-base-formula
+        - https://github.com/elifesciences/api-dummy-formula
+    vagrant:
+        ram: 2048
+        ports:
+            1246: 80
+            1247: 8080
+    aws:
+        type: t2.medium # 4 GB of RAM: Unsure memory stats for PDF generation.
+        ports:
+            - 22
+            - 80:
+                cidr-ip: 10.0.0.0/16 # internal access only
```

Comment:
As far as I am aware just now its serving S3 links for the Emails. I did hear something about a choice page though, so this may change in the future